### PR TITLE
Add support for twofactor_webauthn

### PR DESCRIPTION
**Pull request by st3iny**
I tested it locally and it worked fine.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ TALK_BRANCH
 TASKS_BRANCH
 TEXT_BRANCH
 VIEWER_BRANCH
-ZIPPER_BRANCH
```
Comment (szaimen) [inline readme.md:118]: Why was this removed? Otherwise fine by me

Comment (st3iny) [reply readme.md:118]: Oops :sweat_smile: I pushed a fix.

**Review by szaimen — APPROVED** [review]
LGTM
